llvm/test/CodeGen/DirectX/ContainerData/RootSignature-MultipleEntryFunctions-Error.ll

```diff
@@ -0,0 +1,26 @@
+; RUN: not llc %s --filetype=obj -o -
+; expected-error@-1: More than one entry function defined
```

Comment:
If I remove the expected-error's for the tests in this directory then the tests still pass.  I also note that none of the other tests in the llvm/test/CodeGen directory contained expected-error.  I'm not sure that this is testing what you think it is testing.